bundles/org.openhab.binding.ism8/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -21,6 +21,10 @@
 	<channel-type id="switch-rw">
 		<item-type>Switch</item-type>
 		<label>Digital DataPoint</label>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
+		</tags>
```

Comment:
not the code owner for this one, just user....
I am wondering if a `switch-r` should be tagged `Status` `Mode`.
How do you handle those?